standalone/runtime/tests/test_fee_pool.rs

```diff
@@ -122,6 +122,47 @@ fn get_vault_collateral(vault_id: &VaultId) -> Amount<Runtime> {
         .unwrap()
 }
 
+fn estimate_vault_reward_rate(vault_id: VaultId) -> Result<UnsignedFixedPoint, DispatchError> {
```

Comment:
Have you added this function to the integration test to check that the logic in the RPCs works? Guess we really need to refactor the common logic, starting to look scary to maintain

Reply:
Yea I will refactor in a follow-up pr